mixins/localize/README.md

```diff
@@ -144,7 +93,13 @@ Remember that `<strong>` is for content of greater importance (browsers show thi
 
 Similarly `<em>` *emphasizes* a particular piece of text (browsers show this visually using italics), whereas `<i>` only italicizes the text visually without emphasis.
 
-### Links
+```json
+{
+  "myTerm": "This is <b>bold</b> but <em>not</em> all that <strong>important</strong>."
+};
```

Comment:
No semicolon in JSON:
```suggestion
```json
{
  "myTerm": "This is <b>bold</b> but <em>not</em> all that <strong>important</strong>."
}
```

But maybe better as:
```suggestion
```javascript
{
  myTerm: "This is <b>bold</b> but <em>not</em> all that <strong>important</strong>."
};
```